components/common/circular_chart/circular_chart.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import './circular_chart.scss';
+
+type CircularChartProps = {
+    value: number;
+    isPercentage: boolean;
+    width: number;
+    height: number;
+    type: 'info' | 'warning' | 'error' | 'success';
+};
+
+const CircularChart = ({
+    value,
+    isPercentage,
+    width,
+    height,
+    type,
+}: CircularChartProps): JSX.Element | null => {
+    return (
+        <div className='CircularChart'>
+            <svg
+                viewBox='0 0 36 36'
+                className={`circular-chart ${type}`}
+                width={width ? width.toString() : '36'}
+                height={height ? height.toString() : '36'}
```

Comment:
this is abolutely nitpicking, so dont feel obligated to even react to it! :D :P

`width` or `height` equaling `0` would be falsey, even if it is intended, so it would be better to check for the actual value of it: `width >= 0`

Reply:
lol, the extra-validation doesn't hurt ;)